2019-12-28 00:37:37,354 ERROR: Exception on / [GET] [in /home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/aarti/Desktop/microblog/app/main/routes.py", line 46, in index
    return render_template('index.html', title=_('Home'), form=form, posts=posts.items, next_url=next_url, prev_url=prev_url)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/aarti/Desktop/microblog/app/templates/base.html", line 38, in template
    {% else %}
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/home/aarti/Desktop/microblog/app/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/aarti/Desktop/microblog/app/templates/base.html", line 38, in template
    {% else %}
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 223, in parse_if
    drop_needle=True)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "/home/aarti/.virtualenvs/microblogEnv/lib/python3.6/site-packages/jinja2/parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'else'. Jinja was looking for the following tags: 'endif'. The innermost block that needs to be closed is 'if'.
